tests/test_lsst.py

```diff
@@ -796,22 +796,29 @@ def test_passing_camera_by_hand(self):
             wcs1 = LsstWCS(galsim.CelestialCoord(0.0*galsim.degrees, 0.0*galsim.degrees),
                            self.rotation, chip_name='R:0,1 S:1,1', camera=self.wcs.camera)
 
-        self.assertEqual(str(ww[0].args[0]),
-                         "The camera you passed to LsstWCS does not have the same\n"
-                         "pointing and rotation angle as you asked for for this WCS.\n"
-                         "LsstWCS is creating a new camera with the pointing and\n"
-                         "rotation angle you specified in the constructor for LsstWCS.")
+        expected_message = "The camera you passed to LsstWCS does not have the same\n" \
+                           "pointing and rotation angle as you asked for for this WCS.\n" \
+                           "LsstWCS is creating a new camera with the pointing and\n" \
+                           "rotation angle you specified in the constructor for LsstWCS."
 
+        if hasattr(ww[0], 'args'):
+            self.assertEqual(str(ww[0].args[0]), expected_message)
+        else:
+            self.assertEqual(str(ww[0].message), expected_message)
```

Comment:
I think this is my error.  The exceptions changed from allowing `message` in python 2 to requiring the use of `args` in python 3.  But the warning, as far as I can tell, still use `message` in python 3.  So I think this can just use the message check, without the args part.